Documentation/gettingstarted/kubectl-status.rst

```diff
@@ -0,0 +1,19 @@
+You can monitor as Cilium and all required components are being installed:
+
+.. parsed-literal::
```

Comment:
This one (and the one below) do not need to be `parsed-literal`s.

Reply:
Fixed in latest push.